design/account.go

```diff
@@ -168,6 +168,9 @@ var identityDataAttributes = a.Type("IdentityDataAttributes", func() {
 	a.Attribute("bio", d.String, "The bio")
 	a.Attribute("url", d.String, "The url")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
+	a.Attribute("ContextInformation", a.HashOf(d.String, d.Any), func() {
```

Comment:
> In your example its "ContextInformation": {}
But should be "contextInformation": {}, rihgt?

Correct!

Here you go :)

https://github.com/almighty/almighty-core/pull/1007/commits/96815610e46dd3d25b76abe6293ca2fce92c20b4

( the commit message is wrong, please ignore :D )